spyder/app/mainwindow.py

```diff
@@ -205,25 +167,21 @@
 from spyder.api.exceptions import SpyderAPIError
 from spyder.api.plugins import Plugins, SpyderPluginV2, SpyderDockablePlugin
 
-#==============================================================================
+# ============================================================================
 # Third-party library imports
-#==============================================================================
+# ============================================================================
 import qdarkstyle
 
-#==============================================================================
-# Get the cwd before initializing WorkingDirectory, which sets it to the one
-# used in the last session
-#==============================================================================
-CWD = getcwd_or_home()
+# ============================================================================
+# Get the cwd before initializing WorkingDirectory, which sets it to the
+# one used in the last session

```

Comment:
Please change this block comment to

    Keep a reference to the current working directory.

Reply:
Ok